packages/storybook/src/nimble/select/select.stories.ts

```diff
@@ -191,8 +194,14 @@ const metadata: Meta<SelectArgs> = {
         },
         dropDownPosition: {
             name: 'position',
-            options: ['above', 'below'],
-            control: { type: 'select' },
+            options: [undefined, ...Object.values(DropdownPosition)],
```

Comment:
Like with the combobox stories, we weren't providing a way to demo the default (unset) `dropDownPosition` behavior.